rclrs_example_msgs/msg/VariousTypes.msg

```diff
@@ -0,0 +1,34 @@
+# Primitive types
+bool bool_member true
+int8 int8_member 1
+uint8 uint8_member 2
+byte byte_member 3
+float32 float32_member 1e-2
+
+# Array/sequence of primitive type
+float32[3] float_array [1.0, 2.0, 3.0]
+float32[<=3] float_seq_bounded [4.0, 5.0]
+float32[] float_seq_unbounded [6.0]
+
+# String types
+string string_member "Χαίρετε 你好"
+wstring wstring_member "αντίο σου 再见"
+string<=3 bounded_string_member "äöü"
```

Comment:
I was surprised by this. The length of a string member seems to be measured in unicode code points or something, not bytes ("äöü" has six bytes). I still need to find the exact criterion and replicate it in `BoundedString`.

Reply:
A string. For the wstring, it would be expected that it can fit "äöü".

Reply:
Sending "🔥🔥🔥" (one unicode code point per emoji, 4 bytes each in UTF-8) works fine.

When I send "🤷🏻‍♂️🤷🏻‍♂️🤷🏻‍♂️" (has more than one unicode code point per emoji) or "abcd" in a bounded string, `ros2 topic echo` doesn't receive the message anymore, but the `rclrs` node still does. I'm surprised there is no lower-level check about the length of the string, seem like it's the client library's job.

I changed it to `chars().count()` since that's what mirrors the above behavior and Python's `len()` function for strings.

Reply:
`chars().count()` makes sense to me